sonar-batch/src/main/java/org/sonar/batch/bootstrapper/Batch.java

```diff
@@ -100,15 +100,37 @@ public synchronized Batch start() {
    * @since 4.4
    */
   public Batch executeTask(Map<String, String> analysisProperties, Object... components) {
```

Comment:
@henryju should this be deprecated? That was my plan, otherwise I remove executeTask(Map)